packages/medusa/src/services/line-item.ts

```diff
@@ -182,6 +182,101 @@ class LineItemService extends TransactionBaseService {
     )
   }
 
+  /**
+   * Generate a single or multiple line item without persisting the data into the db.
+   *
+   * NOTE: this currently work with IsolatedProductModule flag
```

Comment:
suggestion: rm, this method can be used no matter if the flag is enabled or not in fact

Reply:
true, meant to emphasize that it is now only used for IsolatedProductModule purposes